generator/test_case_generator.go

```diff
@@ -121,6 +123,8 @@ var testTypeToTestConfig = map[string][]testConfig{
 		{testDir: "../../test/performance/system"},
 		{testDir: "../../test/performance/statsd"},
 		{testDir: "../../test/performance/collectd"},
+		{testDir: "../../test/performance/trace/xray",
+			runMockServer: true},
```

Comment:
Check formatting here. I am not convinced that the linter will want this on a new line